interfaces/builtin/locale_control.go

```diff
@@ -42,6 +69,7 @@ func init() {
 		name:                  "locale-control",
 		summary:               localeControlSummary,
 		implicitOnClassic:     true,
+		implicitOnCore:        release.OnCoreDesktop,
```

Comment:
@robert-ancell is this the best way to achieve this?

Reply:
I think so? systemd is provided by core, so it needs to expose this interface. The question is why this isn't already implicit on core - are you supposed to use another method for changing locale, i.e. only the system can do it?

Reply:
> 

Notice the TODO that it won't work  until snappy exposes it... sounds antique

Reply:
Maybe it has to do with /etc/default/locale not being in writable...

Reply:
I commented in the upstream the changes done to make this work... let's see what they say. https://github.com/snapcore/snapd/pull/13027#issuecomment-1665373891